src/Transition.js

```diff
@@ -258,10 +258,19 @@ class Transition extends React.Component {
   }
 
   safeSetState(nextState, callback) {
+    // We need to track pending updates for instances where a cWRP fires quickly
```

Comment:
Is this a React 16 thing? How can a cWRP fire before cDM flushes?

Reply:
its happening on v15 for me. I don't know. I'd imagine that's not behavior you want but it definitely seems to be happening

Reply:
weird